src/sentry/conf/server.py

```diff
@@ -823,6 +832,17 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     },
 }
 
+if SILO_MODE is None or SILO_MODE in [SiloMode.MONOLITH, SiloMode.CONTROL]:
```

Comment:
Currently, a new mapping is always created when changing org slugs. I think we'd either want to change that behavior to an update in monolith mode, or run the repair task in monolith mode to release no longer used slugs.

Reply:
@mikejihbe Any preference on this? ^

Reply:
We chatted offline about keeping the repair task in monolith mode for now. It can potentially be limited to the control silo if the outbox runs in monolith mode and handles: 1) verifying mappings after changing org slugs, and 2) deleting org mappings on organization delete.